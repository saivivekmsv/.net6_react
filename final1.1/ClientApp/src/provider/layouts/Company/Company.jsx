import React from "react";
import { CompanyContainer } from "../../containers";
import { withAppLayout } from "../../hoc";

const Company = (props) => {
  return <CompanyContainer {...props} />;
};

export default withAppLayout(Company);
