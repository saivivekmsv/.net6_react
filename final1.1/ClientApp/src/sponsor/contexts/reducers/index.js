export * from "./app-layout";
export * from "./manage-company";
export * from "./create-plan";
export * from "./manage-eligibility";
export * from "./manage-census";
export * from "./manage-payroll";
export * from "./manage-maintenance";
export * from "./manage-report";
export * from "./manage-mapper";
