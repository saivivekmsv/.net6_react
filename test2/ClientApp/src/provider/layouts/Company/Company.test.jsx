import React from "react";
import { BrowserRouter as Router } from "react-router-dom";
import { render } from "@testing-library/react";
import { AppLayoutProvider } from "../../contexts";
import Company from "./Company";

test("renders learn react link", () => {
  const wrapper = render(
    <AppLayoutProvider>
      <Router>
        <Company />
      </Router>
    </AppLayoutProvider>
  );
  expect(wrapper.baseElement).toMatchSnapshot();
});
