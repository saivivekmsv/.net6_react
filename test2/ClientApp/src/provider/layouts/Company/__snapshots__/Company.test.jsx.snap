// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<body>
  <div>
    <div
      class="sidebar-container "
    >
      <div
        class="sidebar-placeholder"
      />
      <div
        class="sidebar-overlay"
      />
      <div
        class="sidebar"
      >
        <div
          class="d-flex sidebar-content flex-column justify-content-between py-4 px-3"
        >
          <div
            class="minimized-content col"
          >
            <span
              class="core-icon white"
            >
              <img
                class=""
                src="/assets/icons/svg/core-white.svg"
              />
            </span>
          </div>
          <div
            class="maximized-content col"
          >
            <div>
              <span
                class="core-icon"
              >
                <img
                  class=""
                  height="100%"
                  src="/assets/icons/svg/core.svg"
                  width="60px"
                />
              </span>
            </div>
          </div>
          <div
            class="maximized-content  h-1 col"
          >
            <div
              class="mt-auto"
            >
              <ul>
                <li>
                  <a
                    href="#"
                    target="_blank"
                  >
                    Reference & Enquiry
                  </a>
                </li>
                <li>
                  <a
                    href="#"
                    target="_blank"
                  >
                    Privacy Policy
                  </a>
                </li>
              </ul>
              <br />
              <br />
              <br />
              <div>
                © Congruent Solutions Inc.
              </div>
               
              <div>
                All Rights Reserved
              </div>
              <br />
              <br />
              <br />
              <span
                class="congruent-icon"
              >
                <img
                  class=""
                  src="/assets/icons/svg/congruent-full.svg"
                />
              </span>
              <br />
              <br />
              <br />
              <br />
            </div>
          </div>
        </div>
      </div>
      <div
        class="sidebar-app"
      >
        <nav
          class="header-nav navbar navbar-expand-md navbar-light bg-light sticky-top"
        >
          <div
            class="container-fluid"
          >
            <div
              class="header-nav-row align-items-center row"
            >
              <div
                class="col-2"
              >
                <span
                  class="navbar-brand"
                >
                  <a
                    href="/home"
                  >
                    <img
                      class=""
                      height="31"
                      src="/assets/icons/svg/core.svg"
                    />
                  </a>
                </span>
              </div>
              <div
                class="col-6"
              >
                <div
                  class="navbar-nav"
                >
                  <a
                    href="/home"
                  >
                    <span
                      class="nav-link"
                    >
                      Home
                    </span>
                  </a>
                  <a
                    href="/dashboard"
                  >
                    <span
                      class="nav-link"
                    >
                      Dashboard
                    </span>
                  </a>
                </div>
              </div>
              <div
                class="d-flex justify-content-end col-4"
              >
                <span
                  class="icon-container"
                >
                  <span
                    class="icon"
                  >
                    <svg>
                      user-icon.svg
                    </svg>
                  </span>
                </span>
                <span
                  class="icon-container"
                >
                  <span
                    class="icon"
                  >
                    <svg>
                      vertical-triple-dots.svg
                    </svg>
                  </span>
                </span>
              </div>
            </div>
          </div>
        </nav>
        <div
          class="app-layout container-fluid"
        >
          <div
            class="company-container d-flex flex-column"
          >
            <div
              class="company-heading"
            >
              Manage Company
            </div>
            <div
              class="row"
            >
              <div
                class="col"
              >
                <form
                  class=""
                >
                  <div
                    class="input-group"
                  >
                    <div
                      class="input-group-prepend"
                    >
                      <span
                        class="company-search-button input-group-text"
                      >
                        <img
                          class=""
                          src="/assets/icons/svg/search.svg"
                        />
                      </span>
                    </div>
                    <div
                      class="form-control-seach-text"
                    >
                      <input
                        class="company-search-box form-control form-control-lg"
                        id="company-search-box"
                        placeholder="Search by company name or sponsoring organization"
                        type="search"
                      />
                      <div
                        class="close-button-wrapper "
                      >
                        <button
                          class="close-button"
                          type="button"
                        >
                          <svg
                            aria-hidden="true"
                            class="svg-inline--fa fa-times fa-w-10 fa-2x "
                            data-icon="times"
                            data-prefix="fal"
                            focusable="false"
                            role="img"
                            viewBox="0 0 320 512"
                            xmlns="http://www.w3.org/2000/svg"
                          >
                            <path
                              d="M193.94 256L296.5 153.44l21.15-21.15c3.12-3.12 3.12-8.19 0-11.31l-22.63-22.63c-3.12-3.12-8.19-3.12-11.31 0L160 222.06 36.29 98.34c-3.12-3.12-8.19-3.12-11.31 0L2.34 120.97c-3.12 3.12-3.12 8.19 0 11.31L126.06 256 2.34 379.71c-3.12 3.12-3.12 8.19 0 11.31l22.63 22.63c3.12 3.12 8.19 3.12 11.31 0L160 289.94 262.56 392.5l21.15 21.15c3.12 3.12 8.19 3.12 11.31 0l22.63-22.63c3.12-3.12 3.12-8.19 0-11.31L193.94 256z"
                              fill="currentColor"
                            />
                          </svg>
                        </button>
                      </div>
                    </div>
                  </div>
                </form>
              </div>
            </div>
            <div
              class="align-items-center row"
            >
              <div
                class="my-4 col-5"
              >
                <div
                  class="chip-button-group d-flex align-items-center"
                >
                  <button
                    class="chip-buttons selected btn btn-primary"
                    id="chip-buttons-null"
                    type="button"
                  >
                    All
                  </button>
                  <button
                    class="chip-buttons  btn btn-primary"
                    id="chip-buttons-0"
                    type="button"
                  >
                    Regular
                  </button>
                  <button
                    class="chip-buttons  btn btn-primary"
                    id="chip-buttons-1"
                    type="button"
                  >
                    MEP
                  </button>
                  <button
                    class="chip-buttons  btn btn-primary"
                    id="chip-buttons-2"
                    type="button"
                  >
                    PEP
                  </button>
                  <button
                    class="chip-buttons  btn btn-primary"
                    id="chip-buttons-3"
                    type="button"
                  >
                    Sponsoring Org.
                  </button>
                </div>
              </div>
              <div
                class="col"
              >
                <div
                  class="text-right"
                >
                  0 Companies
                     
                  <a
                    href="/manage-company"
                  >
                    <button
                      class="add-icon"
                    >
                      <svg
                        aria-hidden="true"
                        class="svg-inline--fa fa-plus-square fa-w-14 fa-lg "
                        data-icon="plus-square"
                        data-prefix="fal"
                        focusable="false"
                        role="img"
                        viewBox="0 0 448 512"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <path
                          d="M400 64c8.8 0 16 7.2 16 16v352c0 8.8-7.2 16-16 16H48c-8.8 0-16-7.2-16-16V80c0-8.8 7.2-16 16-16h352m0-32H48C21.5 32 0 53.5 0 80v352c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48V80c0-26.5-21.5-48-48-48zm-60 206h-98v-98c0-6.6-5.4-12-12-12h-12c-6.6 0-12 5.4-12 12v98h-98c-6.6 0-12 5.4-12 12v12c0 6.6 5.4 12 12 12h98v98c0 6.6 5.4 12 12 12h12c6.6 0 12-5.4 12-12v-98h98c6.6 0 12-5.4 12-12v-12c0-6.6-5.4-12-12-12z"
                          fill="currentColor"
                        />
                      </svg>
                    </button>
                  </a>
                </div>
              </div>
            </div>
            <div
              class="loader-wrapper "
            >
              <div
                class="loader-wrapper-overlay"
              >
                <div
                  class="spinner-row d-flex align-items-center"
                >
                  <div
                    class="spinner-border"
                    role="status"
                  >
                    <span
                      class="sr-only"
                    >
                      Loading...
                    </span>
                  </div>
                    
                  Loading
                </div>
              </div>
              <div
                class="d-flex flex-column cspl-table undefined"
              >
                <div
                  class="d-flex table-head align-items-center undefined"
                >
                  <div
                    class="d-flex table-row undefined"
                  >
                    <div
                      class="d-flex table-head-cell column-company-name sortable "
                    >
                      Company Name
                    </div>
                    <div
                      class="d-flex table-head-cell column-company-type sortable "
                    >
                      Type
                    </div>
                    <div
                      class="d-flex table-head-cell column-company-sponsoring sortable "
                    >
                      Sponsoring Org.
                    </div>
                    <div
                      class="d-flex table-head-cell column-company-plan sortable "
                    >
                      Plan IDs
                    </div>
                    <div
                      class="d-flex table-head-cell column-company-employees sortable "
                    >
                      Employees
                    </div>
                    <div
                      class="d-flex table-head-cell column-company-participation sortable "
                    >
                      Participation %
                    </div>
                  </div>
                </div>
                <div
                  class="d-flex flex-column table-body undefined"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</body>
`;
